apps/worker/src/app/telemetry/usecases/userInfoService.usecase.ts

```diff
@@ -2,22 +2,22 @@ import { HttpService } from '@nestjs/axios';
 import { Injectable } from '@nestjs/common';
 import { Cron, CronExpression } from '@nestjs/schedule';
 import {
+  CommunityUserRepository,
+  CommunityOrganizationRepository,
   IntegrationRepository,
   NotificationRepository,
   NotificationTemplateRepository,
-  OrganizationRepository,
   SubscriberRepository,
   TopicRepository,
-  UserRepository,
 } from '@novu/dal';
 import { loadOrCreateMachineId } from '../utils/machine.utils';
 import { sendDataToNovuTrace } from '../utils/sendDataToNovuTrace.utils';
 
 @Injectable()
 export class UserInfoService {
   constructor(
-    private readonly userRepository: UserRepository,
-    private readonly organizationRepository: OrganizationRepository,
```

Comment:
This service is used for telemetry, which counts data in local mongodb, therefore we can use community repository versions directly.

`UserRepository` and `OrganizationRepository` are being resolved conditionally based on if the app is in enterprise version or not. In this case we always know we want to use non-enterprise repositories.